cluster-autoscaler/core/scaledown/nodeevaltracker/max_node_skip_eval_time.go

```diff
@@ -65,6 +71,6 @@ func (l *MaxNodeSkipEvalTime) Update(nodeNames []string, currentTime time.Time)
 	l.lastEvalTime = currentTime
 	minimumTime := l.getMin()
 	longestDuration := currentTime.Sub(minimumTime)
```

Comment:
nit (not really related to current change) - is this metric WAI? It seems that we'll always report a non-zero value, even if there are zero nodes passed here. And if we don't call `Update()` at all when there are no unneeded nodes, we'll end up reporting the last meaningful value indefinitely, which is arguably worse. Maybe I'm missing sth here?

Feel free to disregard (or add a TODO if you agree).